### ISSUE TITLE
[8.0] When you send an e-mail from Odoo with is the correct language: user (who sends) or partner (who receicves)

### ISSUE THREAD

**Issue by rafaelbn** [OCA/social#26]
Hi,

I know this is an Odoo issue, we opened here: https://github.com/odoo/odoo/issues/7896. But Odoo answers

> Hello Rafael,
> We transmit your query to our usability team and they will look into your query.
> As of now we can't do anything else to change this behaviour, only we can send your request to usability team for improvement that's all we can do due to our stable policy.
> Thank you.

This means: OCA is agree to fix this in OCB, OR OCA accept in this project a module which FIX this.

Resume:
- If you send a quote, sale order or an invoice, it will be sent in the language of the partner
- [TOFIX] If you send an e-mail to a Partner through the chatter in the partner's form it will send an e-mail with user's signature in the language of the user an not of the partner.
- [TOFIX] If you select multiple partners and go to "more" -> mass mailing selected partners you will also send the e-mails in the language of the user an not of the parter

Basically (big error of Odoo in my opinion), no matter you translate e-mails templates or anything they will be sent in user's languages and not in partner's languages.
- https://github.com/OCA/OCB/pull/330 is related.
- This module https://github.com/OCA/social/tree/8.0/mail_compose_select_lang doesn't fix the problem as its functionality is other

So OCA, what do you think?

Thanks

**Comment by rafaelbn** [comment]
I forgot that https://github.com/OCA/OCB/pull/330 is related.

**Comment by dreispt** [comment]
If you are able to write a fix, you can do a PR to Odoo master and another one to OCB 8.0.
IMHO there's a good chance for it to be accepted in OCB.

**Comment by rafaelbn** [comment]
Maybe also related: https://github.com/yelizariev/addons-yelizariev/issues/88

OK @dreispt we will write a fix. Thanks

**Comment by rafaelbn** [comment]
Fixed in https://github.com/acsone/mozaik/commit/09dbc54a2c2d87b21c2316118bb17d535a056600
Commented in: https://github.com/odoo/odoo/issues/7896
